cpp/models/ode_secirvvs/parameters_io.h

```diff
@@ -753,636 +835,82 @@ IOResult<void> set_vaccination_data(std::vector<Model<FP>>& model, const std::st
     * @param divi_data_path path to divi data file
     * @param confirmed_cases_path path to confirmed cases file
     * @param population_data_path path to population data file
+    * @param set_vaccination_data boolean to set vaccination data
```

Comment:
To save one input parameter you could also set vaccination_data_path to an empty string per default and check if the string is empty instead of using set_vaccination data.